modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java

```diff
@@ -286,15 +301,67 @@ private List<ObjectViewSortColumn> _addObjectViewSortColumns(
 			});
 	}
 
-	private void _validate(
+	private void _validate(long objectViewId, long objectDefinitionId)
```

Comment:
Could you rename to "_validateDefaultObjectView", please ?

Reply:
ok